bmw_cd_exporter.py

```diff
@@ -113,25 +133,36 @@ def parse_args():
             exit(1)
         if args.interval < 1:
             args.interval = 1
-
-        # register & start prometheus exporter server
-        print('INFO: Registering Prometheus metrics...')
+        print(time.asctime(time.localtime(time.time())),":" 'INFO: Parse JSON File')
```

Comment:
Probably don't need the timestamp here

Reply:
i've used this for logging. I need to update